inference-engine/tests/functional/inference_engine/transformations/batch_to_space_fusion.cpp

```diff
@@ -127,7 +127,7 @@ TEST_F(TransformationTestsF, NegativeBatchToSpaceFusionInvalidMode) {
 
 TEST_F(TransformationTestsF, NegativeBatchToSpaceFusionInvalidRank) {
     {
-        auto data = std::make_shared<opset6::Parameter>(element::f32, Shape{12, 3, 4, 8, 8});
+        auto data = std::make_shared<opset6::Parameter>(element::f32, Shape{16, 3, 4, 8, 8});
```

Comment:
It is  by the depth_to_space bug in master. In the original test, 12 % (block_size*block_size*block_size) != 0, which can pass because of bug in the master.  below is link:
https://github.com/openvinotoolkit/openvino/blob/ca0cdb896f7a4d7019422897690b7c81cc508a75/src/core/src/op/depth_to_space.cpp#L67